ocs_ci/helpers/helpers.py

```
@@ -7874,3 +7875,220 @@ def verify_file_ownership(pod_obj, file_path, expected_uid, expected_gid):
+    )
+    try:
+        exec_cmd(f"oc apply -f {mco_url}")
+    except CommandFailed as e:
+        raise CommandFailed(f"Scale operator deployment failed: {e}") from e
+
+    # 2. Check and Create Entitlement Secret
+    secret_name = "ibm-entitlement-key"
+    secret_ocp = OCP(kind="Secret", namespace=namespace)
+    try:
+        secret_ocp.get(resource_name=secret_name)
+        logger.info(f"Secret '{secret_name}' already exists.")
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)